test/fields/collections/Row/index.ts

```diff
@@ -47,6 +47,35 @@ const RowFields: CollectionConfig = {
         },
       ],
     },
+    {
+      type: 'row',
+      fields: [
+        {
+          name: 'field_with_width_30_percent',
+          label: 'Field with 30% width',
+          type: 'text',
+          admin: {
+            width: '30%',
+          },
+        },
+        {
+          name: 'field_with_width_60_percent',
+          label: 'Field with 60% width',
+          type: 'text',
+          admin: {
+            width: '60%',
+          },
+        },
+        {
+          name: 'field_with_width_20_percent',
+          label: 'Field with 20% width',
+          type: 'text',
+          admin: {
+            width: '20%',
+          },
+        },
+      ],
+    },
```

Comment:
add some new fields for a test case where we want to have a 3rd row wrapping onto the next line, whilst preserving the correct widths